hibernate-core/src/main/java/org/hibernate/Session.java

```diff
@@ -970,6 +970,26 @@ public interface Session extends SharedSessionContract, EntityManager {
 	 */
 	void clear();
 
+	/**
+	 * Return the persistent instances of the given entity class with the given identifiers
+	 * as a list. The position of an instance in the list matches the position of its identifier
+	 * in the given array, and the list contains a null value if there is no persistent instance
+	 * matching a given identifier. If an instance is already associated with the session, that
+	 * instance is returned. This method never returns an uninitialized instance.
+	 * <p>
+	 * Every object returned by {@code findAll()} is either an unproxied instance of the given
+	 * entity class, or a fully-fetched proxy object.
+	 *
+	 * @param entityType the entity type
+	 * @param ids the identifiers
+	 *
+	 * @return an ordered list of persistent instances, with null elements representing missing
+	 *         entities
+	 *
+	 * @since 6.5
+	 */
+	<E> List<E> findAll(Class<E> entityType, Object... ids);
```

Comment:
I'd prefer to see a Javadoc link to `#byMultipleIds` here explaining to use that for more options.

Reply:
Yeah I was going to do that but forgot.